easybuild/easyconfigs/b/BLAST+/BLAST+-2.7.1_static_boost.patch

```diff
@@ -0,0 +1,105 @@
+diff -Nru ncbi-blast-2.7.1+-src.orig/c++/src/build-system/configure.ac ncbi-blast-2.7.1+-src/c++/src/build-system/configure.ac
```

Comment:
@Serpens Please include some comments above this line, to explain what this patch does (and why) + mention author or reference to source of this patch.